README.mdx

```diff
@@ -0,0 +1,206 @@
+to leverage their types and functionalities.
+
+Less duplicate code, easier to bootstrap attention and adoption, and users benefit from the utility you built into 
+something they already presumably own and love - win-win-win!
+
+# Define our Base NFT - KittyVerse
+
+## NonFungibleToken Standard
+
+- Building on the existing standard
+- Encouraged to also implement MetadataViews and ViewResolver contract interfaces, but keeping things simple
+- We're defining the KittyVerse NFT for the purposes of this tutorial, but just as easily could've used an existing NFT 
+project since we're building with attachments
+- If totally new to this, take a look at NonFungibleToken tutorials - TODO: Link these
+
+## Contract Overview
+
+- Contract interface implementation
+    - Events
+    - Nested resource definitions
+- Attribute overview
+- Opportunities to extend the contract
+
+## Deploy KittyVerse contract
+
+- Deploy NonFungibleToken to `0x01`
+- Deploy KittyVerse to `0x02`
+
+## Setup account & Mint KittyVerse NFT
+
+- send `tx_01_setup_kittyverse_account.cdc` signing with `0x02`
+- send `tx_02_mint_kittyverse_nft.cdc` signing with `0x02`
+- send `tx_02_mint_kittyverse_nft.cdc` again signing with `0x02`
+- query NFT.id - execute `script_01_get_kittyverse_ids.cdc` providing `0x02` as argument
+    - see two unique NFT IDS - take note of them, next let's prep them some hats
+
+# Define our Attachments - KittyHats
+
+## Attachments
+
+- What are they?
+    - In Cadence
+    - Simple mental model
+        - "Like glue between resources"
+        - ![KittyVerse & KittyHats attached](./diagrams/cat_with_hat.png)
+- Why do they matter?
+    - Can create attachments for anything without needing original developer's:
+        - Participation
+        - Permission
+
+### Overview
+
+- Conceptual
+    - Attachments are a special type that can be added to resources (doc link needed)
+    - You might be tempted to simply define all your functionality in the attachment, but it's important to know that
+      attachments are not resources.
+        - You'll lose the existence and uniqueness guarantees inherent to resources as well as make access control
+          significantly harder for yourself
+        - As mentioned earlier, attachments are best thought of as glue between resources
+            - Define an attachment that can hold some resource, designed to be added to a base resource
+    - Note that you need access to the base resource to add an attachment to it
+        - So to add an attachment to an NFT, you need to first withdraw that NFT from its Collection
+- Syntax
+    - Accessing an attachment
+    ```cadence
+    let attachmentRef: &KittyHats.HatAttachment? = nft[KittyHats.HatAttachment]
+    ```
+    - Adding an attachment
+    ```cadence
+    access(self) fun addAttachment(toNFT: @KittyVerse.NFT): @KittyVerse.NFT {
+        // If attachment already exists, return
+        if toNFT[HatAttachment] != nil {
```

Comment:
My assumption is that you are going to break it down this little bit. However, it is a readable code but newcomers may not understand the syntax `toNFT[HatAttachment]` as it only apply to attachments.